### ISSUE TITLE
can it run on windows?

### ISSUE THREAD

**Issue by ForbiddenManiac** [codyd51/offkeyboard#1]
can this script run on windows? if yes, then how?

**Comment by gigawasian** [comment]
yes please add this info to the readme.  Ive been trying to get it to work on windows and I'm getting errors :(

**Comment by luiscarlos2000** [comment]
I also need that information right now! I just try to run the py file but no window. Plus, I know only basic python, and I want to control the pc with my synth.

**Comment by SanicStudios** [comment]
Same. Would quite like a way to run this on windows

**Comment by luiscarlos2000** [comment]
Not only on Windows but also on Linux. I don't know how linux users react to this when learning to program...
